File: Assignment40/Java/Program2.java
// PS - write java program which accepts string from user and display below pattern
// h	e	l	l	o	
// h	e	l	l	
// h	e	l	
// h	e	
// h

import java.util.*;

class StrPattern
{
    public void CharPattern(String str)
    {
        char Arr[] = str.toCharArray();

        int len = Arr.length;

        for(int iCnt = 0; iCnt < Arr.length; iCnt++)
        {
            for(int i = 0; i < len; i++)
            {
                System.out.print(Arr[i]+ "\t");
            }
            len--;
            System.out.println();
        }
    }
}

class Program2
{
    public static void main(String arg[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter String");
        String str = sobj.nextLine();

        StrPattern pobj = new StrPattern();

        pobj.CharPattern(str);
    }
}

File: Assignment53/C/Program2.c
// PS - Write a recursive program display below pattern
//1    2    3   4   5

#include <stdio.h>
#include <stdlib.h>

void DisplayR(int iNo)
{
    static int iCnt= 1;
    if(iCnt <= iNo)
    {
        printf("%d \t", iCnt);
        iCnt++;
        DisplayR(iNo);
    }
}

int main()
{
    int iValue = 0;
    printf("Enter number \n");
    scanf("%d", &iValue);

    DisplayR(iValue);

    return 0;
}